docs/deployments/patterns/rollbacks/dotnet-windows-rollbacks/index.md

```diff
@@ -0,0 +1,166 @@
+- Website
+
+Rolling back a database is out of the scope of this guide.  As stated in this [article](https://octopus.com/blog/database-rollbacks-pitfalls), rolling back a database schema change could result in wrong or deleted data.  This guide focuses on scenarios where there were no database changes or the database changes are backward compatible.  Because the database changes are out of scope for rollbacks, the database packages will be "skipped" during the rollback process.
+
+## Existing Deployment Process
+
+For this guide, we will start with the following deployment process for the OctoFX application:
+
+1. Run Database Creation Runbook
+1. Deploy the OctoFX database
+1. Deploy the OctoFX Windows Service
+1. Deploy the OctoFX website
+1. Verify the application
+1. Notify stakeholders
+
+![original windows deployment process](images/original-windows-deployment-process.png)
+
+View the deployment process on [samples instance](https://samples.octopus.app/app#/Spaces-762/projects/01-octofx-original/deployments/process).  Please login as a guest.
+
+## Zero Configuration Rollback
+!include <zero-configuration-rollback>
+
+## Simple Rollback Process
+
+The typical rollback strategy is to skip specific steps and run additional ones during a rollback for most rollbacks.  In this example, the database steps will be skipped with another step to [prevent that release from progressing](/docs/releases/prevent-release-progression.md) will run during a rollback.
+
+The updated deployment process will be:
+
+1. Check Deployment Mode
+1. Run Database Creation Runbook (skip during rollback)
+1. Deploy the OctoFX Database (skip during rollback)
+1. Deploy the OctoFX Windows Service
+1. Deploy the OctoFX Website
+1. Block Release Progression (only run during rollback)
+1. Verify the Application
+1. Notify stakeholders
+
+![simple rollback for windows deployment](images/windows-simple-rollback-process.png)
+
+View the deployment process on [samples instance](https://samples.octopus.app/app#/Spaces-762/projects/02-octofx-simple-rollback/deployments/process).  Please login as a guest.
+
+### Calculate Deployment Mode
+
+!include <calculate-deployment-mode>
+
+### Skip Database Deployment Steps
+
+The two steps related to database deployments, Run Database Creation Runbook and Deploy OctoFX Database, should be skipped during a rollback.  Unlike code, databases cannot easily be rolled back without risking data loss.  For most rollbacks, you won't have database changes.  However, a rollback could accidentally be trigger with a database change.  For example, rolling back a change in **Test** to unblock the QA team.   Skipping these steps during the rollback reduces the chance of accidental data loss.  
+
+To skip these steps during a rollback, set the variable run condition to be:
+
+```
+#{Octopus.Action[Calculate Deployment Mode].Output.RunOnDeploy}
+```
+
+We also recommend adding or updating the notes field to indicate it will only run on deployments.
```

Comment:
```suggestion
:::hint
**Tips:** Remember to replace `Database.Step` with the name of your step. We also recommend adding or updating the notes field to indicate it will only run on deployments.
:::
```